src/main/java/com/service/TwitterImpl.java

```diff
@@ -87,4 +88,40 @@ public Status sendTweets(String tweet) {
         }
         return twitList;
     }
+    public List<TwitterResponse> getTweetBasedOnMyFilter(String tweet)
+    {
+        String twitterHandle;
+        String name;
+        String message;
+        String profileImageUrl;
+        Date created =null;
```

Comment:
Declare the variables where u are using.

Reply:
declared where i am using